templates/zerver/accounts_home.html

```diff
@@ -32,39 +32,46 @@ <h1 class="get-started">{{ _("Sign up for Zulip") }}</h1>
             </div>
             {% endif %}
             <div class="right-side">
-                <form class="form-inline" id="send_confirm" name="email_form"
-                    action="{{ current_url() }}" method="post">
-                    {{ csrf_input }}
-
-                    <div class="input-box no-validate">
-                        <input type="email" id="email" class="email" name="email" value="" required />
-                        <label>Email</label>
-                        <div class="required"></div>
-                        <img class="valid" src="/static/images/checkbox-valid.svg" />
+                {% if no_auth_enabled %}
+                    <div class="alert">
+                        No Authentication Backend is enabled. Please read <a href="http://zulip.readthedocs.io/en/latest/prod-install.html#step-3-configure-zulip">the configuration help</a> to set up an authentication backend.
                     </div>
+                {% else %}
```

Comment:
Our typical template indentation style is to not indent the interior of an if statement unless it's nested right inside another.  I think it's worth changing this to match that style, in part because it mean we don't need to reindent basically anything in this PR.